src/components/TechnologiesList.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <div>
+        <g-link
+            v-for="(tech, index) in allTechnologies"
+            :key="`technologies-list-item-${index}`"
+            :to="`technologies/${tech.node.slug}`"
+            class="about-technologie-item"
+        >
+            <g-image
+                class="technologies-list-item-coverImage"
+                :src="tech.node.coverImage"
+                alt=""
+            />
+            <h2 class="technologies-list-item-title">
+                {{ tech.node.title }}
+            </h2>
+        </g-link>
+    </div>
+</template>
+
+<script>
+
+export default {
+    computed: {
+        allTechnologies() {
```

Comment:
Get technologies from a prop instead of accessing a global object: https://vuejs.org/v2/guide/components.html#Passing-Data-to-Child-Components-with-Props

Reply:
Done